.stripesclirc.js

```diff
@@ -0,0 +1,46 @@
+const webpack = require('webpack');
+
+const environment = process.env.NODE_ENV;
+
+function mirage(config, enabled = false) {
+  if (enabled) {
+    console.info('Using Mirage Server'); // eslint-disable-line no-console
+    return Object.assign({}, config, {
```

Comment:
```suggestion

    return Object.assign({}, config, {
```
Please add empty line after const and before return. Check other places.